cmd/utils/flags.go

```diff
@@ -1600,6 +1606,19 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 			cfg.EthDiscoveryURLs = SplitAndTrim(urls)
 		}
 	}
+	if ctx.GlobalIsSet(HTTPRpcTimeoutFlag.Name) {
+		arg := ctx.GlobalString(HTTPRpcTimeoutFlag.Name)
+		_, err := strconv.Atoi(arg)
+		if err == nil {
+			arg = arg + "ns"
+		}
```

Comment:
Why parse, it's a Duration type already ?

Reply:
Yeah, I cleaned this up and added some tests.